docs/src/_changelog.md

```diff
@@ -29,6 +35,19 @@ Changelog](https://keepachangelog.com).
   fixed ([#1202]).
 
 ### Changed
+- Removed `Pkg.build("IJulia")` support. IJulia no longer uses a
+  build step for kernel installation. The build-time configuration system has
+  been replaced with runtime functions. Use [`installkernel()`](@ref) to install
+  or update kernels, and [`update_jupyter_path()`](@ref) to configure the
+  Jupyter executable path.
+- IJulia now uses [Scratch.jl](https://github.com/JuliaPackaging/Scratch.jl) to
+  store configuration preferences instead of writing to `DEPOT_PATH/prefs/`.
```

Comment:
Why not Preferences.jl?

Reply:
Wouldn't that make changing these settings force re-precompilation? I don't think we need that.